patches/minecraft/net/minecraft/world/spawner/CatSpawner.java.patch

```diff
@@ -0,0 +1,10 @@
+--- a/net/minecraft/world/spawner/CatSpawner.java
++++ b/net/minecraft/world/spawner/CatSpawner.java
+@@ -80,6 +80,7 @@
+       if (catentity == null) {
+          return 0;
+       } else {
++         if(net.minecraftforge.common.ForgeHooks.canEntitySpawn(catentity, p_221122_2_, p_221122_1_.func_177958_n(), p_221122_1_.func_177956_o(), p_221122_1_.func_177952_p(), null, SpawnReason.NATURAL) == -1) return 0;
```

Comment:
There's lots of other calculations in this class that are used to determine where to spawn a Cat. I didn't want to make a large patch to bypass that code, so I just stuck it in the `spawnCat` method.